src/s2/s2polyline.cc

```diff
@@ -50,6 +50,13 @@ static const unsigned char kCurrentLosslessEncodingVersionNumber = 1;
 S2Polyline::S2Polyline()
   : s2debug_override_(S2Debug::ALLOW) {}
 
+S2Polyline::S2Polyline(S2Polyline&& other)
```

Comment:
Does this have anything to do with Windows?

Reply:
No, will separate into another PR. It was to silence a warning.